setup.py

```diff
@@ -50,19 +53,21 @@ def finalize_options(self):
         pass
 
     def run(self):
-        # Windows versions of etag do not seem to expand wildcards (which Unix shells normally
-        # do for Unix utilities), so find all of the files ourselves.
-        files = [join('src', f) for f in os.listdir('src') if f.endswith(('.h', '.cpp'))]
+        # Windows versions of etag do not seem to expand wildcards (which Unix shells normally do for Unix utilities),
+        # so find all of the files ourselves.
+        files = [ join('src', f) for f in os.listdir('src') if f.endswith(('.h', '.cpp')) ]
         cmd = 'etags %s' % ' '.join(files)
         return os.system(cmd)
 
 
+
 def main():
+
     version_str, version = get_version()
 
     settings = get_compiler_settings(version_str)
 
-    files = [relpath(join('src', f)) for f in os.listdir('src') if f.endswith('.cpp')]
+    files = [ relpath(join('src', f)) for f in os.listdir('src') if f.endswith('.cpp') ]
```

Comment:
Wait - this may be because it just needs to be rebased with master.  I *recently* cleaned up lint / flake8 warnings.